exchanges/okx/okx_wrapper.go

```diff
@@ -1453,7 +1453,8 @@ func (ok *Okx) GetAvailableTransferChains(ctx context.Context, cryptocurrency cu
 	}
 	chains := make([]string, 0, len(currencyChains))
 	for x := range currencyChains {
-		if !cryptocurrency.IsEmpty() && !strings.EqualFold(cryptocurrency.String(), currencyChains[x].Currency) {
+		if (!cryptocurrency.IsEmpty() && !strings.EqualFold(cryptocurrency.String(), currencyChains[x].Currency)) ||
+			(!currencyChains[x].CanDeposit && !currencyChains[x].CanWithdraw) {
```

Comment:
`[ERROR] | LOG | 01/05/2023 13:37:54 | Okx failed to get cryptocurrency deposit address for BTC [chain BTC-Lightning]. Err: specified chain BTC-Lightning not found`

```suggestion
		if (!cryptocurrency.IsEmpty() && !strings.EqualFold(cryptocurrency.String(), currencyChains[x].Currency)) ||
			(!currencyChains[x].CanDeposit && !currencyChains[x].CanWithdraw) ||
			!currencyChains[x].MainNet {
```